src/components/AddressFormFields/index.tsx

```diff
@@ -97,8 +112,8 @@ const AddressForm = ({ type, errorMessages }: AddressFormProps) => {
             })}
             defaultValue={formValues[`${type}-city`]}
           />
-        </div>
-        <div style={{ flex: "1" }}>
+        </DSFormField>
+        <DSFormField>
```

Comment:
Good point for this and the other similar comment. These updates are out of scoped for this PR -- I don't want to change functionality too much since it's a DS update. I'll create tickets for this, however.